lib/utils/transformSelector.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+"use strict";
+
+const selectorParser = require("postcss-selector-parser");
+
+module.exports = function(
+  result /*: Object*/,
+  node /*: Object*/,
+  cb /*: Function*/
+) {
+  try {
+    return selectorParser(cb).processSync(node, { updateSelector: true });
```

Comment:
Why `updateSelector: true` is needed here? It's `true` by default. https://github.com/postcss/postcss-selector-parser/blame/v3.1.1/API.md#L678-L680

Reply:
Does this really block PR?

Reply:
It's a new utility which might work the same as existing utility. It would create more confusion. So I wouldn't merge just yet.

Reply:
It doesn't look like `updateSelector` option has default value `true` (probably documentation issue).
https://github.com/postcss/postcss-selector-parser/blob/v3.1.1/src/processor.js#L65
Also, check dts file:
https://github.com/postcss/postcss-selector-parser/blob/v3.1.1/postcss-selector-parser.d.ts#L89

Reply:
You're right. Looks like behavior changed, but docs weren't updated.